src/App.jsx

```diff
@@ -47,17 +28,22 @@ function Itemslist(props) {
 Itemslist.propTypes = {
   className: PropTypes.string.isRequired,
   list: PropTypes.arrayOf(PropTypes.string).isRequired,
+  handelDelete: PropTypes.func.isRequired,
 };
 
 function App() {
-  const [itemsarray, setItemsarray] = useState([]);
+  const [itemsArray, setitemsArray] = useState([]);
   const [inputvalue, setInputvalue] = useState('');
 
+  const handelDelete = (id) => {
+    const newList = [...itemsArray].filter((item) => item.id !== id);
+    setitemsArray(newList);
+  };
   const handleChange = (e) => {
     setInputvalue(e.target.value);
   };
   const handleSubmit = () => {
-    setItemsarray([...itemsarray, inputvalue]);
+    setitemsArray([...itemsArray, { id: Date.now(), task: inputvalue }]);
```

Comment:
Do not use `Date.now()` as an identifier, use `uuid` library.

Reply:
In order to absolutely ensure that there would never be duplicate ids in all scenarios, some actual identifier that doesn't depend on one variable like time, but on a combination of variables, like time, system, network, etc.